### ISSUE TITLE
Word definition

### ISSUE THREAD

**Issue by rkhoury18** [LangProc/langproc-2021-lab#8]
Can a word be a mix of upper case and lower case letters?

**Comment by ymherklotz** [comment]
Yes, it does sound ambiguous in the spec though.  You can always have a look at the reference output though in the `test/` folder.

**Comment by johnwickerson** [comment]
> A word is any contiguous sequence of lower-case or upper-case alphabetic characters.

**Comment by johnwickerson** [comment]
As opposed to:

> A word is a sequence of lower-case alphabetic characters or a sequence of upper-case alphabetic characters.
